front/app/containers/SearchPage/SearchPage.tsx

```diff
@@ -502,9 +502,9 @@ class SearchPage extends React.Component<SearchPageProps, SearchPageState> {
   };
 
   componentDidMount() {
-    let searchTerm = new URLSearchParams(this.props.location.search).getAll("q")
-    if (URLSearchParams(this.props.location.search).has("q")) {
-      let q = { key: 'AND', children: [{ children: [], key: searchTerm.toString() }] };
+    let searchTerm = new URLSearchParams(this.props.location.search)
+    if (searchTerm.has('q')) {
```

Comment:
is this more so what you where suggesting? @ericharding

Reply:
Yea, that looks a lot better.